src/sentry/static/sentry/app/components/events/interfaces/spans/span_tree.tsx

```diff
@@ -0,0 +1,665 @@
+import React from 'react';
+import styled from 'react-emotion';
+import _ from 'lodash';
+
+import space from 'app/styles/space';
+import Count from 'app/components/count';
+
+import {SpanType, SpanEntry, SentryEvent} from './types';
```

Comment:
File names are generally camelBack in sentry's javascript.

Reply:
done.